src/components/List/ListItem.tsx

```diff
@@ -112,15 +114,21 @@ class ListItem extends UIComponent<any, any> {
           debug={debug}
           // disappearing={!truncateHeader}
           truncateMain={truncateHeader}
-          rootCSS={isHovering && { color: 'inherit' }}
+          rootCSS={selection && isHovering ? { color: 'inherit' } : undefined}
```

Comment:
Thanks for these updates.  The List component was copied directly form a prototype we had on the side.  It needs a cleanup round for styling and behavior.  You can ignore that for now and I'll take a pass at it soon.  The props and patterns for styling it will change quite a bit.

Reply:
This is OK as-is.  No need to revert 😃